packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,476 @@
+| size               | `small`, `medium`, `large`              | `medium`      | Defines size of the Breadcrumb                                                           |
+| truncateNameLength |                                         |               | Amount of symbols for truncated name                                                     |
+
+### BreadcrumbItem
+
+#### Anatomy
+
+![visual anatomy of the BreadcrumbItem component](./assets/breadcrumb-item-anatomy.png)
+
+BreadcrumbItem can be:
+
+- Button
+- Link
+- Non-clickable content
+- Dropdown Menu
+- Icon (optional)
+
+It can contain a tooltip.
+
+#### DOM
+
+Non-clickable element
+
+```HTML
+<li>
+  {children}
+</li>
+```
+
+Link
+
+```HTML
+<li>
+  <a href="#">
+    {children}
+  </a>
+</li>
+```
+
+Button
+
+```HTML
+<li>
+  <button>
+    {children}
+  </button>
+</li>
+```
+
+Usage
+
+```jsx
+<BreadcrumbItem>
+  Item 1
+</BreadcrumbItem>
+<BreadcrumbItem as="button" onClick={() => console.log('smth...')}>
+  Item 2
+</BreadcrumbItem>
+<BreadcrumbItem as="a" href="#">
+  Item 3
+</BreadcrumbItem>
+```
+
+#### API
+
+| Property     | Values                                  | Default   | Purpose                                          |
+| ------------ | --------------------------------------- | --------- | ------------------------------------------------ |
+| as           | `a`, `button`                           | `button`  | Defines what component to render, button or link |
+| active       | boolean                                 | false     | Indicates if the item is active                  |
+| disabled     | boolean                                 | false     | Disables Breadcrumb item                         |
+| focusMode    | `off`, `no-tab`, `tab-exit`, `tab-only` | `off`     | Sets the focus behavior for the Breadcrumb.      |
+| icon         | _slot_                                  |           | Sets icon                                        |
+| iconPosition | `before`, `after`                       | `before`  | Sets icon position                               |
+| tooltop      | _slot_                                  | undefined | Sets custom tooltip                              |
+
+#### Breadcrumb icon
+
+An icon can be SVG or image.
+The icon might be added as a prop:
+
+```jsx
+<BreadcrumbItem icon={<IconComponent />}>Item 1</BreadcrumbItem>
+<BreadcrumbItem icon="../path/name.png">Item 1</BreadcrumbItem>
+<BreadcrumbItem>
+  <SomeSVG />
+  Item 1
+</BreadcrumbItem>
+```
+
+### BreadcrumbDivider
+
+#### DOM
+
+```HTML
+<li aria-hidden="true">
+  {children}
+</li>
+```
+
+#### API
+
+| Property | Values                               | Default   | Purpose              |
+| -------- | ------------------------------------ | --------- | -------------------- |
+| divider  | `chevron`, `slash`, custom component | `chevron` | Sets type of divider |
+
+### BreadcrumbDropdown
```

Comment:
Since this is using `react-menu` under the hood is `BreadcrumbMenu` a better name?

Reply:
I think yes. Thanks

Reply:
Fixed the name, resolving this,